server.js

```diff
@@ -1,16 +0,0 @@
-const express = require("express");
-const app = express();
-const port = 3000;
-const studentRoutes = require("./src/students/routes");
-
```

Comment:
How to have simple username password based signup process here.

Only details required
Username
Password
Name
DOB